enatega-multivendor-admin/src/components/Coupon/Coupon.jsx

```diff
@@ -152,6 +152,7 @@ function Category(props) {
                   onBlur={event => {
                     onBlur(discountErrorSetter, 'discount', event.target.value)
                   }}
+                  inputProps={{ min: "0" }} // Add min attribute to restrict negative values
```

Comment:
Min check is already added in constraints. We don't need to specify here.

Reply:
No,  before I made changes. It was taking negative values using arrows but not after the changes. I think the min condition you are mentioning is for the red border maybe ,when the values are negative in the discount field.

Reply:
ok sure